### PR TITLE
network: refactoring network monitoring

### PR DESCRIPTION
**Pull request by pillip8282**
previous network monitoring uses lwIP api directly. So it can not be
used on loadable configuration.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [713a8bb2d6ef7d938dfa9a19d4ac58d8a7c0b41f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [713a8bb2d6ef7d938dfa9a19d4ac58d8a7c0b41f] - Code Rule Check Result:
apps/system/utils/netcmd_netmon.c:63: ERROR: [IDT_M_TAB] please, no space before tabs
apps/system/utils/netcmd_netmon.c:65: ERROR: [IDT_M_TAB] please, no space before tabs
apps/system/utils/netcmd_netmon.c:69: ERROR: [IDT_M_TAB] please, no space before tabs
apps/system/utils/netcmd_netmon.c:71: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:458: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:459: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:461: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:463: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:464: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:465: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:467: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:468: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:469: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:470: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:471: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:472: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:473: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:474: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:475: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:476: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:477: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:479: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:480: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:481: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:482: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:483: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:484: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:486: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:487: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:488: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:490: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:491: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:492: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:493: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:494: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:495: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:496: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:497: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:498: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:499: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:500: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:501: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:502: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:503: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:504: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:505: ERROR: [IDT_M_TAB] please, no space before tabs
os/net/netmgr/netstack_lwip.c:531: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

**Comment by seinfra** [comment]
Target : [1b6ad0cdfbfce2d83bb6ad13791d716971a65155] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1b6ad0cdfbfce2d83bb6ad13791d716971a65155] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [f2b1a3f28a46067c4bc530cc21aadce9d822b620] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f2b1a3f28a46067c4bc530cc21aadce9d822b620] - Code Rule Check OK.

File: apps/system/utils/netcmd_netmon.c [hunk 1]
```diff
@@ -44,75 +44,63 @@
...
-				printf("%-8s", "*");						\
-			} else {										\
-				printf("%-8d", sock->remote_port);			\
-			}												\
-		}													\
-	} while (0)
+static inline void PRINT_IPV(struct netmon_sock *sock)
+{
+	if (sock->type & NETMON_TYPE_IPV6) {
+		printf("IPv6 (not yet supported)\t");
```
Comment (sangwon03) [inline apps/system/utils/netcmd_netmon.c]: Why did you remove to print information about IPv6?

Comment (pillip8282) [reply apps/system/utils/netcmd_netmon.c]: monitoring IPv6 can't be provided with current bsd socket. it can be added after bsd socket API supports ipv6
